base/setup/usetup/lang/en-US.h

```diff
@@ -139,50 +139,56 @@ static MUI_ENTRY enUSIntroPageEntries[] =
     {
         6,
         8,
-        "ReactOS Setup is in an early development phase. It does not yet",
-        TEXT_STYLE_NORMAL
+        "ReactOS Version Status",
+        TEXT_STYLE_HIGHLIGHT
     },
     {
         6,
-        9,
-        "support all the functions of a fully usable setup application.",
+        11,
+        "ReactOS is in Alpha stage, meaning it is not feature-complete",
         TEXT_STYLE_NORMAL
     },
     {
         6,
         12,
-        "The following limitations apply:",
+        "and under heavy development. It is recommended to use only for",
```

Comment:
"It is recommended to use" *it* "only for..." <-- I think it's missing a "it" before "only" ?